src/components/NcModal/NcModal.vue

```diff
@@ -714,10 +722,13 @@ export default {
 				allowOutsideClick: true,
 				fallbackFocus: contentContainer,
 				trapStack: getTrapStack(),
+				// Esc can be used without stop in content or additionalTrapElements where it should not deacxtivate modal's focus trap.
+				// Focus trap is deactivated on modal close anyway.
+				escapeDeactivates: false,
 			}
 
 			// Init focus trap
-			this.focusTrap = createFocusTrap(contentContainer, options)
+			this.focusTrap = createFocusTrap([contentContainer, ...this.additionalTrapElements], options)
```

Comment:
Making the `additionalTrapElements` watcher `immediate` probably also works but on init definitely seems more intuitive

Reply:
It won't work. Watcher with `immediate: true` runs on an instance creation before rendering, while the focus trap must be initialized after the modal is actually mounted and shown (opened).